<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentWindow</class>
 <widget class="QMainWindow" name="StudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="studentTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>1021</width>
      <height>411</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>ID#</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>First Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Last Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Year Level</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Gender</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Program</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>510</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pageNum">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>510</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>|&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jump">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>550</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jump</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="viewPrograms"/>
    <addaction name="viewColleges"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="addStudent"/>
    <addaction name="deleteStudent"/>
    <addaction name="editStudent"/>
    <addaction name="refreshTable"/>
   </widget>
   <widget class="QMenu" name="menuSort">
    <property name="title">
     <string>Sort</string>
    </property>
    <addaction name="sortID"/>
    <addaction name="sortFirstName"/>
    <addaction name="sortLastName"/>
    <addaction name="sortYearLevel"/>
    <addaction name="sortGender"/>
    <addaction name="sortProgram"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuManage"/>
   <addaction name="menuSort"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="viewPrograms">
   <property name="text">
    <string>Programs</string>
   </property>
  </action>
  <action name="viewColleges">
   <property name="text">
    <string>Colleges</string>
   </property>
  </action>
  <action name="addStudent">
   <property name="text">
    <string>Add Student</string>
   </property>
  </action>
  <action name="deleteStudent">
   <property name="text">
    <string>Delete Student</string>
   </property>
  </action>
  <action name="editStudent">
   <property name="text">
    <string>Edit Student</string>
   </property>
  </action>
  <action name="sortID">
   <property name="text">
    <string>ID#</string>
   </property>
  </action>
  <action name="sortFirstName">
   <property name="text">
    <string>First Name</string>
   </property>
  </action>
  <action name="sortLastName">
   <property name="text">
    <string>Last Name</string>
   </property>
  </action>
  <action name="sortYearLevel">
   <property name="text">
    <string>Year Level</string>
   </property>
  </action>
  <action name="sortGender">
   <property name="text">
    <string>Gender</string>
   </property>
  </action>
  <action name="sortProgram">
   <property name="text">
    <string>Program</string>
   </property>
  </action>
  <action name="refreshTable">
   <property name="text">
    <string>Refresh Table</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
